api/envoy/config/listener/v3/listener_components.proto

```diff
@@ -332,13 +333,12 @@ message ListenerFilter {
 
   // The name of the filter to instantiate. The name must match a
   // :ref:`supported filter <config_listener_filters>`.
-  // [#extension-category: envoy.transport_sockets.downstream]
   string name = 1 [(validate.rules).string = {min_len: 1}];
 
-  // Filter specific configuration which depends on the filter being instantiated.
-  // See the supported filters for further documentation.
-  // [#extension-category: envoy.filters.listener]
   oneof config_type {
+    // Filter specific configuration which depends on the filter being
```

Comment:
in this case it was necessary to move the comment inside the `oneof` (as is done with others)

not doing so threw a formatting error - so we might need to look at the `\n` mangling in protodoc to make it less brittle